android/src/main/java/com/swmansion/reanimated/nodes/PropsNode.java

```diff
@@ -25,6 +25,28 @@
   private final JavaOnlyMap mPropMap;
   private final ReactStylesDiffMap mDiffMap;
 
+  private void addProp(WritableMap propMap, String key, Object value) {
```

Comment:
```suggestion
  private static void addProp(WritableMap propMap, String key, Object value) {
```

Reply:
The PR looks good, I'm just going to commit my suggestion and merge it afterwards

Reply:
OMG, I can't commit it apparently, because the PR is from your fork